### ISSUE TITLE
Parameters for input reduction

### ISSUE THREAD

**Issue by vhavlena** [VeriFIT/kofola#6]
Add parameters for `low/high` reduction of both first/second automaton for inclusion (so-far we have only single parameter for the same reduction of both automata).

**Comment by OndrejAlexaj** [comment]
This is already done, see methods:
`spot::twa_graph_ptr inclusion_check::init_aut_A(const spot::twa_graph_ptr &aut_A)`
and
`cola::tnba_complement inclusion_check::init_compl_aut_b(const spot::twa_graph_ptr &aut_B)`.

**Comment by vhavlena** [comment]
Great. Is it possible to set it using command line arguments?

**Comment by OndrejAlexaj** [comment]
Yes, for example `--params='preproc_incl_A=low;preproc_incl_B=high'`.

**Comment by vhavlena** [comment]
Great. I put them into readme file and close this issue.
